src/Component/Navbar.jsx

```diff
@@ -29,83 +29,133 @@ function Navbar() {
   return (
     <>
       <nav className="Navbar">
-        <div 
+        <div
           className="nav-icons"
-          style={{ cursor: 'pointer',fontSize:'2em' }}
+          style={{ cursor: "pointer", fontSize: "2em" }}
           onClick={toggleDarkMode}>
           {slidebarClick ? <GiCrossMark /> : <FaRegSun />}
         </div>
-        <div className="left-sight" onClick={handleRedirect} style={{textAlign:'center',fontSize:'2em',cursor:'pointer'}} 
-        >
-          CareerZunction</div>
+        <div
+          className="left-sight"
+          onClick={handleRedirect}
+          style={{ textAlign: "center", fontSize: "2em", cursor: "pointer" }}>
+          CareerZunction
+        </div>
         <div className="right-sight">
           <div className="cart-item">
-            <Link to="/" className='link'>Home</Link>
+            <Link to="/" className="link">
+              {" "}
+              <i class="fa-solid fa-house"></i> Home
+            </Link>
           </div>
           <div className="cart-item">
-            <Link to="/intern" className='link'> Internships</Link>
+            <Link to="/intern" className="link">
+              {" "}
```

Comment:
why you have added an icon on the navbar

Reply:
sorry , I have edited icons very before and forget to remove. That's why this happend

Reply:
ok